charts/internal/cert-management-garden-runtime/values.yaml

```diff
@@ -0,0 +1,87 @@
+# SPDX-FileCopyrightText: 2019 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+nameOverride: ""
+fullnameOverride: cert-controller-manager
+
+replicaCount: 1
+
+images:
+  cert-management: any-repo:any-tag
```

Comment:
Again, just for me to understand: I assume the image value is overridden somewhere else?

Reply:
The images are injected from the imagevector. See https://github.com/gardener/gardener-extension-shoot-cert-service/blob/972a04e7384a923ffbe22b0fb7f4df2e407c29bb/pkg/controller/actuator.go#L319